conventions/documentation/quick-starts.md

```diff
@@ -0,0 +1,265 @@
+  * DON’T: *Let’s set up our environment.*
+* Format UI elements accordingly:
+  * **Copy for buttons, dropdowns, tabs, fields, and other UI controls:** Write the copy as it appears in the UI and bold it.
+  * **All other UI elements—including page, window, and panel names:** Write the copy as it appears in the UI and bold it.
+  * **Code or user-entered text:** Use monospaced font
+  * **Hints:** If a hint to a navigation or masthead element is included, style the text as you would a link.
+  * **CLI commands:** Use monospaced font.
+    * In running text, use a bold monospaced font for a command.
+    * If a parameter or option is a variable value, use an italic monospaced font.
+    * Use a bold monospaced font for the parameter and a monospaced font for the option.
+
+### Modals and in-app messaging
+* For confirmation modals included with the instructions, refer to [PatternFly’s modal guidelines](https://www.patternfly.org/v4/components/modal/design-guidelines#get-required-user-input), specifically the “Validate user decisions” section.
+  * **Headline:** A headline is usually phrased as a question. Include important keywords (like “permanent”) in the headline. Avoid extraneous copy, like *Are you sure?*, which distracts users from the main outcome of the action.
+  * **Body text:** Body text gives information about the action’s consequence.
+  * **Buttons:** Buttons allow a user to answer the headline question. These can simply be “yes” or “no,” but adding some context increases clarity—especially when skimming.
+* For error messages, use [PatternFly’s error messaging guidelines](https://www.patternfly.org/v4/ux-writing/error-messages).
+
+### Check your work module
+Once a user completes a step, a Check your work module appears. This module prompts the user to answer a yes/no question about the step results, which gives them the opportunity to review their work. For this module, you’ll only need to write a single yes/no question.
+
+If the user answers Yes, a check mark will appear.
+
+If the user answers No, an error message appears with a link to relevant documentation, if necessary. The user then has the opportunity to go back and try again.
+
+
+## <a name="partiv"></a> Part IV: How can I contribute a quick start?
+
+### Internal contributions
+
+#### Process goals:
+* To provide user-centric, functional quick starts for OpenShift users across a wide range of product areas
+* To create a successful collaboration across teams (UXD, CCS, PM, etc.) for every quick start
+* To define which teams are involved at each step of the quick start effort so that the end result is organized, clean, and in final form for users
+* To respect the bandwidth and input of each team
+
+#### Phase 1: Planning
+1. Determine which product area needs a quick start.
+  * Identify stumbling blocks or complex areas where we can enable user success with as little friction as possible. This information will be gathered from:
+    * UXD user research
```

Comment:
Can we add periods after each of these list items? They create full sentences when combined with the list intro ("This info will be gathered from...")